src/pages/api/login.ts

```diff
@@ -46,10 +46,13 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
                 const nocCode = decodedIdToken['custom:noc'];
                 const schemeOpName = decodedIdToken['custom:schemeOperator'];
                 const schemeOpRegion = decodedIdToken['custom:schemeRegionCode'];
-                if (nocCode && nocCode.split('|').length === 1) {
-                    const operatorName = await getOperatorNameByNocCode(nocCode);
-                    const operatorCookieValue = JSON.stringify({ operator: operatorName, noc: nocCode });
-                    setCookieOnResponseObject(OPERATOR_COOKIE, operatorCookieValue, req, res);
+
+                if (nocCode) {
+                    if (nocCode.split('|').length === 1) {
```

Comment:
Do we need some logic for if the nocCode.split('|').length does not equal 1? It looks like here, it will just not do anything?

Reply:
No, it doesn't need to do anything here, it will perform this logic on the multipleOperators page, we just don't want it getting into the final else if